### ISSUE TITLE
Doesn't work with puppeteer

### ISSUE THREAD

**Issue by deoxykev** [DataHenHQ/till#4]
When I try to use till with curl, it works fine after install the cert, but when I specify the proxy with puppeteer:

```js
(async () => {
  const browser = await puppeteer.launch({
     headless: false,
      args: [
        '--proxy-server=http://localhost:2933',
        '--ignore-certificate-errors',
        '--ignore-certificate-errors-spki-list '
     ],
    });
```

I get a `err_ssl_version_or_cipher_mismatch` error message on Chrome, using Windows 10.

**Comment by paramaw** [comment]
I'm not sure if any of the comments on this issue would help: puppeteer/puppeteer#2377

**Comment by deoxykev** [comment]
I've already tried all the ignoreSSL flags, no luck.

Is there a way to try to run a SOCKS5 proxy?

**Comment by paramaw** [comment]
At the moment we don't support SOCKS5 proxy.

So far there are two things that I've tried that worked for me:
1. The Certificate Authority (CA) certs that TILL generates needs to be installed on the OS
2. I was able to verify manually that my browser (Firefox on macOS) works with Till.


I wasn't able to try puppeteer yet, but in theory, if you're able to install the root CA on the OS, or directly in the headless browser instance, it should work.

So far, the following are link that I've found, but I haven't put further investigation if it solves your issue or not:
https://stackoverflow.com/questions/66956565/installing-cert-for-headless-chrome-in-selenium-docker-image
